app/importers/graph_model_importer.rb

```diff
@@ -63,7 +63,7 @@ def create_citations(graph_work_order, note_id, target_number)
   end
 
   def last_imported_work_order
-    @_last ||= Graph::WorkOrder.profile.where(source: WORK_ORDERS_IMPORT).last&.reference || '00000000'
```

Comment:
the `.last` here was evading the profiling since it's called on the resulting scope rather than the profiler